package.json

```diff
@@ -37,7 +37,7 @@
     "express-bunyan-logger": "1.3.2",
     "fastboot": "github:wikia/fastboot#582162a5e2eb39c2d94f857e53ee0efff4544447",
     "fastboot-app-server": "1.0.1",
-    "jwplayer-fandom": "github:wikia/jwplayer-fandom#1.0.25",
+    "jwplayer-fandom": "github:wikia/jwplayer-fandom#4be7225f36b91d392b3454002222c86e38092b59",
```

Comment:
This is just for testing, we will swap it for a release version when we merge https://github.com/Wikia/jwplayer-fandom/pull/29/files